src/lib/wallets/connecters/binanceW3W/binanceWallet.ts

```diff
@@ -0,0 +1,106 @@
+import type { InjectedConnectorOptions } from "@wagmi/core/connectors/injected";
+import { InjectedConnector } from "wagmi/connectors/injected";
+import { Chain, Wallet, getWalletConnectConnector } from "@rainbow-me/rainbowkit";
+import { getWalletConnectUri } from "../helper";
+import { getHref } from "@binance/w3w-utils";
+
+const isInBinance = () => {
+  try {
+    return (window as any)?.ethereum?.isBinance === true;
+  } catch (error) {
+    return false;
+  }
+};
+
+export interface BinanceW3WOptions {
+  projectId: string;
+  chains: Chain[];
+  walletConnectVersion?: "2";
```

Comment:
just to confirm, the only valid value for `walletConnectVersion` is "2" and non other value can be passed?